src/integrationTest/java/com/mongodb/hibernate/BasicCrudIntegrationTests.java

```diff
@@ -37,11 +37,8 @@
 
 @SessionFactory(exportSchema = false)
 @DomainModel(
-        annotatedClasses = {BasicCrudIntegrationTests.Book.class, BasicCrudIntegrationTests.BookDynamicallyUpdated.class,
-                BasicCrudIntegrationTests.BookCountByAuthor.class, BasicCrudIntegrationTests.BookSummary.class
-        },
-        extraQueryImportClasses = {BasicCrudIntegrationTests.BookCountByAuthor.class}
-        )
+        annotatedClasses = {BasicCrudIntegrationTests.Book.class, BasicCrudIntegrationTests.BookDynamicallyUpdated.class
+        })
```

Comment:
no need to include DTO classes if they don't contain annotations Hibernate recognized; `extraQueryImportClasses` is used to import named queries with annotations on entity classes.